src/XrdHttp/XrdHttpReq.cc

```diff
@@ -1489,8 +1498,9 @@ int XrdHttpReq::ProcessHTTPReq() {
 
 
           if (!prot->Bridge->Run((char *) &xrdreq, 0, 0)) {
-            prot->SendSimpleResp(404, NULL, NULL, (char *) "Could not run close request.", 0, false);
-            return -1;
+            TRACEI(REQ, " Failed to run close request on the bridge.");
```

Comment:
Maybe you can put the error message in a string allowing you to not repeat the same string in the two calls?

Reply:
Done in latest commit.